### DISCUSSION TITLE
Problems with Obsidian git?

### DISCUSSION THREAD

**Discussion by thibaultmol** [vrtmrz/obsidian-livesync#97] [Q&A]
Hi, the Readme mentions that no other sync solutions should be used with livesync. 
But I was planning to use the Obsidian-git plugin as a backup solution (so just FROM Obsidian to a git repository). I'm guessing this would be fine to use in the same vault that livesync was running, right?

**Comment by vrtmrz** [comment]
Thank you for asking me!
I was reminded that there was too less documentation.

The root of this limitation is that files being processed by other solutions may be detected by this plug-in, and it causes file corruption.
This is the possibility, but not every time. And, it can be avoided by thinning detection.

If we use periodic-sync and Batch database update, it can be used with Git. (Actually, I use FolderSync for backup on Android.)

**Reply by Vyeche** [reply]
Is it possible to do a once a day git sync? Then continue replication. Can that be part of a routine?

**Reply by vrtmrz** [reply]
Sorry, I missed this.
This is possible. However, ensure that `Skip patterns` are configured to include `\/.git/`. (Default value is already so)

![image](https://user-images.githubusercontent.com/45774780/189269809-2c8ccf90-e71d-48db-b6d4-2e0d064886cf.png)

**Comment by thibaultmol** [comment]
For my usecase I would have to use live sync in a real-time mode actually

**Reply by vrtmrz** [reply]
It will be ok, if you don't sync `.git` dir by the LiveSync.
And, if you want to be safer, you can temporarily turn the LiveSync off (easy to do with `Ctrl+P` and `Toggle LiveSync`).

**Reply by thibaultmol** [reply]
The plan is to have a sync TO git every 5 minutes or so. So I guess I'll just exclude .git then

**Reply by vrtmrz** [reply]
Yes, I think it will work fine!
